internal/website/content/_index.md

```diff
@@ -33,9 +33,10 @@ To start using the Go CDK, install it using `go get`:
 go get gocloud.dev
 ```
 
-Then follow the [Go CDK
-tutorial](https://github.com/google/go-cloud/tree/master/samples/tutorial).
-Links to additional documentation and samples are available below.
+Then follow the [Go CDK tutorial][]. Links to additional documentation and
+samples are available below and in the site navigation bar.
+
+[Go CDK tutorial]: {{< ref "/tutorials/cli.md" >}}
```

Comment:
I'd suggest calling this tutorial `uploader` to be more in-line with the binary it builds and less tied to the notion of CLI which could be used for other providers etc.

Reply:
I'd like to add a more basic HTTP server tutorial at some point, and I was thinking the progression from command-line to HTTP would make more sense in terms of progression than "Uploader and Web Uploader". WDYT?

Reply:
Fair enough, then maybe `cli-upload` or some such? `cli` could be also for pubsubs, CLIs and what not

Reply:
Done.